### ISSUE TITLE
Link discord account with Facebook account

### ISSUE THREAD

**Issue by stanfear** [miscord/miscord#296]
So basically, I have one channel that is used by a few people, all of which have a discord account. but they use messenger to get the bubbles.

trouble is, one of them never stops playing with nicknames. which makes it hard to follow who's who.

so it would be nice to be able to configure somewhere that this messenger account is this Discord user
that way, maybe the colour of the name, and the discord username could be shown ?

only thing to figure out is whether or not the mention should be relayed to Messenger

**Comment by Bjornskjald** [comment]
@stanfear it's not a great idea, it would need to log in on each of these accounts from one IP address which could trigger auto-bot or auto-spam mechanisms on Facebook

**Comment by Bjornskjald** [comment]
ohh I get it. I don't think you could assign roles to webhooks because there's only 1 webhook per channel

**Comment by fortepc** [comment]
instead of trying to link the accounts, it's much easier to just make your users have their nickname be the same in both places, Ie their Pokemon go username

**Comment by stanfear** [comment]
I have no way of enforcing that, I manage the Discord server but not the messenger discussion, there, people can do anything they want

**Comment by fortepc** [comment]
Ah, well I'd say you really need to get the people who manage the messenger discussion on board. This whole thing works way better when everyone is on the same page.
